chore: isolate custom example app to be injected on matter SDK (#64)
canonical/matter-pi-gpio-commander: app/README.md

# App

> This directory is copied into CHIP project's 
> [examples/lighting-app](https://github.com/project-chip/connectedhomeip/tree/master/examples/lighting-app),
> the reason for that is described on this Chip projects [upstream issue](https://github.com/project-chip/connectedhomeip/issues/32925).

Reviewer: ```suggestion
To build, this directory should be copied into CHIP project's 
[examples](https://github.com/project-chip/connectedhomeip/tree/master/examples) directory.
The reason for that is described [in this issue](https://github.com/project-chip/connectedhomeip/issues/32925).
```
